synfig-studio/src/gui/widgets/widget_coloredit.cpp

```diff
@@ -427,6 +429,23 @@ Widget_ColorEdit::Widget_ColorEdit():
 		//I use Gtk::ColorSelection widget here.
 		hvsColorWidget = manage(new Gtk::ColorSelection());
 		setHVSColor(get_value());
+
+		//navigating to inner implementation color wheel widget
+		std::vector<Widget*> internal_child = hvsColorWidget->get_children();
+		Gtk::Box* box_cast = static_cast<Gtk::Box*>( internal_child[0] );
+		std::vector<Widget*> internal_child_2 = box_cast->get_children();
+		Gtk::Box* box_cast_2 = static_cast<Gtk::Box*>( internal_child_2[0] );
+		std::vector<Widget*> internal_child_3 = box_cast_2->get_children(); //internal_child_3[0] is the color wheel widget
+
+		internal_child_3[0]->signal_button_press_event().connect([&](GdkEventButton *ev){ wheel_pressed=true; escape_cancel=false; return false;},false);
```

Comment:
Try to remove `wheel_pressed` variable - you already know that wheel is pressed if `hvsColorWidget->is_adjusting() == true`

Reply:
done